bulk-export/README.md

```diff
@@ -0,0 +1,31 @@
+# Bulk export of resource assignments for point in time
```

Comment:
question: why is the bulk-export code in this pull-request?

Reply:
FYI: it appears the diff is showing other commits, we should probably try to rebase instead of merge commit to hide these.